src/votekit/election_state.py

```diff
@@ -91,8 +91,9 @@ def round_outcome(self, round: int) -> dict:
         """
         if self.curr_round == round:
             return {
-                "Elected": [c for s in self.elected for c in s],
-                "Eliminated": [c for s in self.eliminated_cands for c in s],
+                "Elected": [s for s in self.elected ],

```

Comment:
nit: I'd probably use list(self.elected) here for clarity and a tiny speed improvement